File: Assignments/Assignment_19/p9.c
#include<stdio.h>
void main(){

        int size,num;
        printf("Enter the size of array: \n");
        scanf("%d",&size);
        int arr[size];
        for(int i=0;i<size;i++){

                printf("Enter the arr element %d :",i+1);
                scanf("%d",&arr[i]);
        }num=arr[0];
        for(int i=1;i<size;i++){

                if(arr[i]>num){

                        num=arr[i];
                }else{

                        printf("This is not an increasing array\n");
                        break;
                }
        }if(num==arr[size-1]) printf("This is an increasing array\n");
}

File: Flutter/Assignment/For Loop-2/p9.dart
import "Dart:io";
void main(){
	print("Enter a number:");
	int? x=int.parse(stdin.readLineSync()!);
	List num=[];
	for(int i=0;i<x;i++){
		int? element=int.parse(stdin.readLineSync()!);
		num.add(element);
	}

	print("Enter a number to search:");
	int? search=int.parse(stdin.readLineSync()!);
	int flag=0;
	for(int i=0;i<x;i++){
		if(search==num[i]){
			flag=1;	
		}
	}
	if(flag==1){
		print("$search exists in the list");
	}else{
		print("$search does not exixt in the list");
	}
}
